tests/components/bluesound/test_init.py

```diff
@@ -0,0 +1,47 @@
+"""Test __init__."""
+
+import asyncio
+
+from pyblu.errors import PlayerUnreachableError
+
+from homeassistant.components.bluesound import async_unload_entry
+from homeassistant.core import HomeAssistant
+
+from .conftest import PlayerMocks
+
+from tests.common import MockConfigEntry
+
+
+async def test_setup_entry(hass: HomeAssistant, setup_config_entry: None) -> None:
+    """Test a successful setup entry."""
+    assert hass.states.get("media_player.player_name1111").state == "playing"
+
+
+async def test_unload_entry(
+    hass: HomeAssistant, setup_config_entry: None, config_entry: MockConfigEntry
+) -> None:
+    """Test entries are unloaded correctly."""
+    assert await async_unload_entry(hass, config_entry)
+    await hass.async_block_till_done()
+
+    assert hass.states.get("media_player.player_name1111").state == "unavailable"
```

Comment:
These can be merged and also use the config entry state

Reply:
Is there anything obvious wrong here? The config_entry is still LOADED after unloading.

Reply:
Feel free to push if you can't seem to solve it

Reply:
I have combined the tests. It is part of `test_setup_entry` now. The `config_entry` is still LOADED at the end of the test.

Reply:
I found the problem.